### PR TITLE
Enhancement/307

### PR DESCRIPTION
**Pull request by theflexbird**
#307

### CODE REVIEW

File: src/client/components/CampaignEditor/CampaignForm.react.js [hunk 1]
```diff
@@ -1,67 +1,90 @@
 import React, { Component, PropTypes } from 'react';
 import { connect } from 'react-redux';
-import { Field } from 'redux-form';
+import { Field, formValueSelector } from 'redux-form';
```
Comment (nczeranka) [inline src/client/components/CampaignEditor/CampaignForm.react.js]: Is it used?

File: src/client/components/CampaignEditor/CampaignForm.react.js [hunk 2]
```diff
@@ -1,67 +1,90 @@
...
         </div>
         <FormFieldError hasError={hasError} error={error} />
       </div>
     );
   }
 }
 
+const CopyField = ({currentService, customerId, campaignId}) => (
+  <span className="input-group">
+    <input type="text" className="form-control" id="link" readOnly value={`${currentService.location}/public/landingpage/c_${customerId}/${campaignId}`} />
```
Comment (nczeranka) [inline src/client/components/CampaignEditor/CampaignForm.react.js]: What is provided by currentService.location? Is it assure that it is the same as in the Consul key/values?

File: src/client/components/CampaignEditor/CampaignForm.react.js (hunk 1 above)
Comment (theflexbird) [reply src/client/components/CampaignEditor/CampaignForm.react.js]: Yes, it is. Look line 13 and 14.

File: src/client/components/CampaignEditor/CampaignForm.react.js (hunk 2 above)
Comment (theflexbird) [reply src/client/components/CampaignEditor/CampaignForm.react.js]: Look at file `src/server/routes/index.js` line 66.

**Pull request merged**